core/src/main/java/org/jruby/RubyArray.java

```diff
@@ -60,28 +60,19 @@
 import org.jruby.runtime.marshal.UnmarshalStream;
 import org.jruby.util.ByteList;
 import org.jruby.util.Pack;
-import org.jruby.util.PerlHash;
 import org.jruby.util.Qsort;
 import org.jruby.util.RecursiveComparator;
-import org.jruby.util.SipHashInline;
 import org.jruby.util.TypeConverter;
 
 import java.io.IOException;
 import java.lang.reflect.Array;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Comparator;
-import java.util.IdentityHashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.ListIterator;
-import java.util.RandomAccess;
+import java.util.*;
```

Comment:
Ah - there's not even any useful changes in this file. I'll just revert it. I had experimented with integrating our pack into JRuby which is why I was in this file, but there is a DSL limitation that makes it hard so I thought I'd leave that until later.